app/src/main/java/org/simple/clinic/scanid/IndiaNHIDGender.kt

```diff
@@ -0,0 +1,54 @@
+package org.simple.clinic.scanid
+
+import android.os.Parcelable
+import androidx.annotation.VisibleForTesting
+import com.squareup.moshi.FromJson
+import com.squareup.moshi.ToJson
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.util.room.SafeEnumTypeAdapter
+
+sealed class IndiaNHIDGender : Parcelable {
```

Comment:
This does not need to be a sealed class with a `SafeEnumTypeAdapter` since this is only necessary for types which are both present in the API *and* are saved in the local database. Since we do not store the `IndiaNHIDPayload` in the database and only consume it at runtime, we can use a regular enum type.